pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1665,7 +1678,7 @@
             :-  'json'
             ~|  [%unexpected-fact-mark p.cage.sign]
             ?>  =(%json p.cage.sign)
-            ;;(json q.q.cage.sign)
+            !<(json q.cage.sign)
```

Comment:
I'm unsure what the practical different between the `;;` and `!<` here is. I'd naively expect the latter to be slower, and the former to be safe in this context? But that might be a false sense of safety.

Reply:
Nevermind, I think I missed this one: https://github.com/urbit/urbit/commit/306dbbaead83ca19fe95c1b60c22820f3591f6dd#r51086544